docs/GettingStarted.md

```diff
@@ -1367,6 +1367,7 @@ content = Net::HTTP.get(URI('http://127.0.0.1/index.html'))
 | `distributed_tracing` | Enables [distributed tracing](#distributed-tracing) | `true` |
 | `service_name` | Service name used for `http` instrumentation | `'net/http'` |
 | `split_by_domain` | Uses the request domain as the service name when set to `true`. | `false` |
+| `response_code_errors` | Range or Array of HTTP status codes that should be traced as errors. | `400...599` |
```

Comment:
I suggest changing this to `error_status_codes`, given "status code" here can only refer to the response (the HTTP request does not have a status code):
```suggestion
| `error_status_codes` | Range or Array of HTTP status codes that should be traced as errors. | `400...599` |
```